### PR TITLE
Improve descriptions

### LINKED ISSUE #3: Improve descriptions
**Motivation**

To have good descriptions for readers.

**Acceptance criteria**

- [x] Useful descriptions.
- [x] Remove marketing taglines.

### PR DESCRIPTION
**Pull request by cuongw**
Close #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,18 +115,17 @@
...
-- [Mailspring](https://github.com/Foundry376/Mailspring) - The best mail client for Mac, Linux, and Windows. 👏
-- [Nylas Mail](https://github.com/nylas/nylas-mail) - Open source, extensible mail client. 👏
+- [Mailspring](https://github.com/Foundry376/Mailspring) - Free, open source, beautiful and fast mail client. 👏
 - [Thunderbird](https://www.thunderbird.net) - Free email application, easy to install and customize.
 
 ## Markdown Editors
 
 - [ghostwriter](https://github.com/wereturtle/ghostwriter) - Cross-platform, aesthetic, distraction-free Markdown editor. 👏
-- [Haroopad](https://github.com/rhiokim/haroopad) - The Next Document processor based on Markdown. 👏
+- [Haroopad](https://github.com/rhiokim/haroopad) - The next document processor based on markdown. 👏
```
Comment (sindresorhus) [inline README.md]:
```suggestion
- [Haroopad](https://github.com/rhiokim/haroopad) - The next document processor based on Markdown. 👏
```

File: README.md [hunk 2]
```diff
@@ -154,7 +153,7 @@
 - [Alacritty](https://github.com/jwilm/alacritty) - Cross-platform, GPU-accelerated terminal emulator. 👏
 - [Hyper](https://github.com/zeit/hyper) - Terminal built on web technologies. 👏
 - [Konsole](https://github.com/KDE/konsole) - Terminal emulator for the K Desktop Environment. 👏
-- [Tilda](https://github.com/lanoxx/tilda) - Gtk based drop down terminal for Linux and Unix. 👏
+- [Tilda](https://github.com/lanoxx/tilda) - Gtk based drop down terminal for Linux. 👏
```
Comment (sindresorhus) [inline README.md]:
```suggestion
- [Tilda](https://github.com/lanoxx/tilda) - GTK based drop down terminal. 👏
```

Linux is implied.

File: README.md [hunk 3]
```diff
@@ -165,22 +164,23 @@
...
 
 - [Aurees](https://aurees.com/) - Easy, fast and free.
 - [CodeReview](https://github.com/FabriceSalvaire/CodeReview) - Git GUI tool to perform code review. 👏
 - [GitAhead](https://github.com/gitahead/gitahead) - Graphical Git client designed to help you understand and manage your source code history. 👏
-- [Git Cola](https://github.com/git-cola/git-cola) - The highly caffeinated Git GUI. 👏
+- [Git Cola](https://github.com/git-cola/git-cola) - Powerful Git GUI with a slick and intuitive user interface. 👏
 - [Git Extensions](https://github.com/gitextensions/gitextensions) - Standalone UI tool for managing git repositories. 👏
 - [Git Horizon](https://www.cycligent.com/git-tool) - Cross-platform, Actionable, Graphical Git Dashboard.
-- [GitKraken](https://www.gitkraken.com/) - The most popular Git GUI for Linux.
+- [GitKraken](https://www.gitkraken.com/) - The legendary Git GUI client for Linux.
```
Comment (sindresorhus) [inline README.md]: Words like `legendary` are useless. It doesn't describe what its strengths are. Also don't use `Linux` in the descriptions.

File: README.md [hunk 4]
```diff
@@ -165,22 +164,23 @@
...
 - [SmartGit](https://www.syntevo.com/smartgit/) - Graphical Git client with support for SVN and Pull Requests for GitHub and Bitbucket.
-- [Sublime Merge](https://www.sublimemerge.com/) - Git Client, done Sublime.
-- [ungit](https://github.com/FredrikNoren/ungit) - The easiest way to use git. 👏
+- [Sublime Merge](https://www.sublimemerge.com/) - Powerful Git client from Sublime Text.
```
Comment (sindresorhus) [inline README.md]:
```suggestion
- [Sublime Merge](https://www.sublimemerge.com/) - Powerful Git client from the creators of Sublime Text.
```

File: README.md [hunk 5]
```diff
@@ -195,8 +195,8 @@
 - [Docker](https://www.docker.com/) - Powerful, performs operating-system-level virtualization.
 - [QEMU](https://github.com/qemu/qemu) - Generic and open source machine emulator and virtualizer. 👏
 - [Vagrant](https://github.com/hashicorp/vagrant) - Tool for building and distributing development environments. 👏
-- [VirtualBox](https://www.virtualbox.org/) - Powerful, brimming with terrific features and, best of all, it’s free.
-- [VMWare Fusion](https://www.vmware.com/) - Powerful, commercial VM developed by VMware.
+- [VirtualBox](https://www.virtualbox.org/) - Free, powerful and brimming with terrific features.
```
Comment (sindresorhus) [inline README.md]: The description here is totally useless and says nothing about what the thing is.

File: README.md [hunk 6]
```diff
@@ -165,22 +164,23 @@
...
+- [Git Cola](https://github.com/git-cola/git-cola) - Powerful Git GUI with a slick and intuitive user interface. 👏
 - [Git Extensions](https://github.com/gitextensions/gitextensions) - Standalone UI tool for managing git repositories. 👏
 - [Git Horizon](https://www.cycligent.com/git-tool) - Cross-platform, Actionable, Graphical Git Dashboard.
-- [GitKraken](https://www.gitkraken.com/) - The most popular Git GUI for Linux.
+- [GitKraken](https://www.gitkraken.com/) - The legendary Git GUI client for Linux.
 - [SmartGit](https://www.syntevo.com/smartgit/) - Graphical Git client with support for SVN and Pull Requests for GitHub and Bitbucket.
-- [Sublime Merge](https://www.sublimemerge.com/) - Git Client, done Sublime.
-- [ungit](https://github.com/FredrikNoren/ungit) - The easiest way to use git. 👏
+- [Sublime Merge](https://www.sublimemerge.com/) - Powerful Git client from Sublime Text.
+- [ungit](https://github.com/FredrikNoren/ungit) - The easiest way to use Git. 👏
```
Comment (sindresorhus) [inline README.md]: This is a marketing tagline and not a useful description.

**Comment by sindresorhus** [comment]
Generally, too many fast, slick, etc words, and too much marketing gibberish and taglines. The descriptions should be objective and useful. They should describe what the thing is and what it's good at (no using generic words like `slick`).

**Comment by cuongw** [comment]
Hi @sindresorhus, I just pushed a new commitment to fix some problems. Please review it again. Thank you.

File: README.md [hunk 7]
```diff
@@ -94,8 +94,8 @@
 - [Android Studio](https://developer.android.com/studio/index.html) - The fastest tools for building apps on every type of Android device.
 - [CLion](https://www.jetbrains.com/clion/) - Cross-platform IDE for C and C++.
 - [DataGrip](https://www.jetbrains.com/datagrip/) - Many databases, one tool.
-- [Eclipse](https://www.eclipse.org/) - The Platform for Open Innovation and Collaboration.
-- [GoLand](https://www.jetbrains.com/go/) - Clever IDE to Go.
+- [Eclipse](https://www.eclipse.org/) - Free and open source and is written by Java and plugin architecture allows Eclipse to be extended in other languages.
```
Comment (sindresorhus) [inline README.md]: Too many `and` words. The sentence could be better written.

File: README.md [hunk 8]
```diff
@@ -94,8 +94,8 @@
...
+- [GoLand](https://www.jetbrains.com/go/) - Cross platform IDE built specially for Go developers.
```
Comment (sindresorhus) [inline README.md]: Cross-platform

File: README.md [hunk 9]
```diff
@@ -121,17 +121,17 @@
 ## Markdown Editors
 
 - [ghostwriter](https://github.com/wereturtle/ghostwriter) - Cross-platform, aesthetic, distraction-free Markdown editor. 👏
-- [Haroopad](https://github.com/rhiokim/haroopad) - The next document processor based on markdown. 👏
+- [Haroopad](https://github.com/rhiokim/haroopad) - The next document processor based on Markdown. 👏
 - [Moeditor](https://github.com/Moeditor/Moeditor) - Your all-purpose markdown editor. 👏
-- [Remarkable](https://github.com/jamiemcg/remarkable) - Powerful markdown editor available for Linux. 👏
+- [Remarkable](https://github.com/jamiemcg/remarkable) - Open source, lightweight and powerful markdown editor. 👏
```
Comment (sindresorhus) [inline README.md]: markdown => Markdown

File: README.md [hunk 10]
```diff
@@ -121,17 +121,17 @@
...
 - [ReText](https://github.com/retext-project/retext) - Simple but powerful editor for Markdown and reStructuredText. 👏
-- [Typora](https://typora.io/) - Simple, beautiful, flexible and powerful markdown editor.
+- [Typora](https://typora.io) - Simple, beautiful, flexible and powerful markdown editor.
```
Comment (sindresorhus) [inline README.md]: markdown => Markdown

File: README.md [hunk 11]
```diff
@@ -121,17 +121,17 @@
...
 
 ## Music
 
 - [Amarok](https://github.com/KDE/amarok) - Powerful music player for Linux with an intuitive interface. 👏
-- [Audacious](https://github.com/jade28/awesome-linux-apps/edit/master/README.md) - Open source media player that’s widely popular among audiophiles. 👏
-- [Clementine](https://github.com/clementine-player/Clementine) - Cross-platform, open source music player that’s a popular choice of Linux-using music enthusiasts. 👏
+- [Audacious](https://audacious-media-player.org/) - Opensource audio player that is low resource and has a minimal feature interface. 👏
```
Comment (sindresorhus) [inline README.md]: Opensource is not a word. And the sentence is not grammatically correct.

File: README.md [hunk 12]
```diff
@@ -121,17 +121,17 @@
...
+- [Clementine](https://github.com/clementine-player/Clementine) - Cross platform music player and library organizer forked from Amarok 1.4. 👏
```
Comment (sindresorhus) [inline README.md]: Cross-platform

File: README.md [hunk 13]
```diff
@@ -153,8 +153,8 @@
 - [Alacritty](https://github.com/jwilm/alacritty) - Cross-platform, GPU-accelerated terminal emulator. 👏
 - [Hyper](https://github.com/zeit/hyper) - Terminal built on web technologies. 👏
 - [Konsole](https://github.com/KDE/konsole) - Terminal emulator for the K Desktop Environment. 👏
-- [Tilda](https://github.com/lanoxx/tilda) - Gtk based drop down terminal for Linux. 👏
-- [Tilix](https://github.com/gnunn1/tilix/) - Tiling terminal emulator for Linux. 👏
+- [Tilda](https://github.com/lanoxx/tilda) - Highly configurable, easy to configure the hotkeys and change the appearance. 👏
```
Comment (sindresorhus) [inline README.md]: Too much "configure"

File: README.md [hunk 14]
```diff
@@ -166,7 +166,7 @@
 - [Oni](https://github.com/onivim/oni) - Modern Modal Editing - powered by Neovim. 👏
 - [Oni2](https://github.com/onivim/oni2) - Native, lightweight modal code editor. 👏
 - [Sublime Text](https://www.sublimetext.com/) - Sophisticated text editor for code, markup and prose.
-- [Vim](https://github.com/vim/vim) - The ubiquitous text editor. 👏
+- [Vim](https://github.com/vim/vim) - Highly configurable text editor for efficiently creating and changing any kind of text. 👏
```
Comment (sindresorhus) [inline README.md]: "any kind of text" sounds weird. All text editors can do that.

File: README.md [hunk 15]
```diff
@@ -177,10 +177,10 @@
 - [Git Cola](https://github.com/git-cola/git-cola) - Powerful Git GUI with a slick and intuitive user interface. 👏
 - [Git Extensions](https://github.com/gitextensions/gitextensions) - Standalone UI tool for managing git repositories. 👏
 - [Git Horizon](https://www.cycligent.com/git-tool) - Cross-platform, Actionable, Graphical Git Dashboard.
-- [GitKraken](https://www.gitkraken.com/) - The legendary Git GUI client for Linux.
+- [GitKraken](https://www.gitkraken.com/) - Git client with efficiency, elegance and reliability.
 - [SmartGit](https://www.syntevo.com/smartgit/) - Graphical Git client with support for SVN and Pull Requests for GitHub and Bitbucket.
-- [Sublime Merge](https://www.sublimemerge.com/) - Powerful Git client from Sublime Text.
-- [ungit](https://github.com/FredrikNoren/ungit) - The easiest way to use Git. 👏
+- [Sublime Merge](https://www.sublimemerge.com/) - Powerful Git client from the creators of Sublime Text.
+- [ungit](https://github.com/FredrikNoren/ungit) - Friendly and web-based interface. 👏
```
Comment (sindresorhus) [inline README.md]: "Interface" for what?

File: README.md [hunk 16]
```diff
@@ -195,8 +195,8 @@
 - [Docker](https://www.docker.com/) - Powerful, performs operating-system-level virtualization.
 - [QEMU](https://github.com/qemu/qemu) - Generic and open source machine emulator and virtualizer. 👏
 - [Vagrant](https://github.com/hashicorp/vagrant) - Tool for building and distributing development environments. 👏
-- [VirtualBox](https://www.virtualbox.org/) - Free, powerful and brimming with terrific features.
-- [VMWare Fusion](https://www.vmware.com/) - Powerful and commercial VM developed by VMware.
+- [Virtual Box](https://www.virtualbox.org/) - Powerful x86 and AMD64/Intel64 virtualization product.
```
Comment (sindresorhus) [inline README.md]: Still not a great description.

**Comment by sindresorhus** [comment]
I'm not sure what more to comment. Most of the descriptions are still pretty generic and not very useful. Way to much "flexible", "configurable", "efficient", "simple" (try to count how many times those words are used...).

**Comment by cuongw** [comment]
@sindresorhus,
I am very happy because you took your time to review my pull request. I try to complete this issue, I hope it pass. Thanks 🙏.

**Comment by sindresorhus** [comment]
I don't have time to do more reviews. Just try to improve all the descriptions.

**Pull request merged**